src/Pages/Main/Components/ProductCard.scss

```
@@ -0,0 +1,70 @@
+@import '../../../Styles/common.scss';
+
+.product-card-container {
+  display: flex;
+  max-width: 1050px;
+  overflow: scroll;
+  .product-card {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    min-width: 245px;
+    padding: 0px 10px;
+    transition: 0.5s;
+    overflow: hidden;
+    .product-link {
```

```suggestion
    overflow: hidden;
    
    .product-link {
```
새로운 셀렉터 앞에서는 한줄 띄워주시면 가독성 차원에서 좋겠네요!

---

넵!!!